OuterWildsRandomSpeedrun/SpawnPointMenu/SpawnPointMenu.cs

```diff
@@ -0,0 +1,29 @@
+using OWML.Common;
+using SpawnPointSelector;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace OuterWildsRandomSpeedrun
+{
+  /// <summary>
+  /// An extension of Menu. In addition to handling some special logic for enabling/disabling
+  /// the menu, this is also used by TitleScreenManagerPatches to identify when certain
+  /// aspects of the title screen should be enabled/disabled.
+  /// </summary>
+  public class SpawnPointMenu : Menu
+  {
+    public override void EnableMenu(bool shouldEnable)

```

Comment:
change shouldEnable to shouldExpand

Reply:
I could, but it's doing a bit more than only expanding it. There's logic inside Menu.EnableMenu() that does a bunch of things like managing sounds to play, selecting items, etc.